hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/LoadBalancer.java

```diff
@@ -156,4 +156,12 @@
    */
   default void updateBalancerLoadInfo(Map<TableName, Map<ServerName, List<RegionInfo>>>
     loadOfAllTable){}
+
+  /**
+   * Check if the ServerName is bogus.
+   * @param sn server name to be checked
+   */
+  default boolean isBogusServerName(ServerName sn) {
```

Comment:
Can be removed once we use `BOGUS_SERVER_NAME` directly in TRSP

Reply:
Ditto.